VERSION

```diff
@@ -1 +1 @@
-2.0.0-dev
\ No newline at end of file
+2.1.0
```

Comment:
```suggestion
2.0.0
```

Reply:
Hey @alannix-lw, I believe when we do a major version bump we usually change VERSION file on this PR. As the release scripts will automatcially increase the minor/patch version based on commit message.  

It might be time to take another look at whether we want to add major versions to the version checking script. We'd always intentionally not tried to automate it. @afiune